models/prices/prices_tokens.sql

```diff
@@ -1776,6 +1778,7 @@ VALUES
     ("nuls-nuls", "polygon", "NULS", "0x8b8e48a8cc52389cd16a162e5d8bd514fabf4ba0", 8),
     ("blank-blockwallet","polygon", "BLANK", "0xf4C83080E80AE530d6f8180572cBbf1Ac9D5d435", 18),
     ("fin-fin", "polygon", "FIN", "0x576c990a8a3e7217122e9973b2230a3be9678e94", 18),
+    -- ("sphere-sphere-finance", "polygon", "SPHERE", "0x62f594339830b90ae4c084ae7d223ffafd9658a7", 18),
```

Comment:
https://api.coinpaprika.com/v1/coins/sphere-sphere-finance
This token is not active, so comment out.

Reply:
Hey @Hosuke, do you expect something from me?

Reply:
No, just explain why this line is in comment, as this token is no longer active in api.

Reply:
No, just explain why this line is in comment, as this token is no longer active in api.

Reply:
Sorry for the broken English.